app/ControllerFunctions/ReadAccessFunctions.php

```diff
@@ -45,7 +41,12 @@ public function __construct(SessionFunctions $sessionFunctions)
 	 */
 	public function album($albumID, bool $obeyHidden = false)
 	{
-		if (in_array($albumID, array('f', 's', 'r', '0'))) {
+		if (in_array($albumID, array(
+			'f',
+			's',
+			'r',
+			'0'
+		))) {
```

Comment:
Is there a reason for laying the array out like this?

Reply:
nah it's just the formatting done by my Phpstorm when I do CTRL+L ^^"